curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/674f06b4b4d6eb4f6139ad61.md

```diff
@@ -0,0 +1,52 @@
+---
+id: 674f06b4b4d6eb4f6139ad61
+title: Task 2
+challengeType: 19
+dashedName: task-2
+---
+
+# --instructions--
+
+This task doesn't have audio. Read the question below and select the correct answer.
+
+# --questions--
+
+## --text--
+
+What might be involved in website tests?
```

Comment:
I considered your suggestion, but since "website tests" is one of the vocabulary items we want to teach in this dialogue, I think it's fine to test understanding of this phrase after teaching the individual words in task 1.